clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -1453,19 +1453,20 @@ public String toHtml(Map<String, String> dynamicUpdateModes) {
         List<ConfigKey> configs = sortedConfigs();
         StringBuilder b = new StringBuilder();
         b.append("<ul class=\"config-list\">\n");
-
         for (ConfigKey key : configs) {
             if (key.internalConfig) {
                 continue;
             }
-            b.append("<li>");
-            b.append("<b>");
-            b.append(key.name);
-            b.append("</b>: ");
-            b.append(key.documentation);
-            b.append("<br/>");
-            // details
-            b.append("<ul class=\"horizontal-list\">");
+            b.append("<li style=\"padding: 15 0 15 20; border-bottom: 1px solid black;\">");
```

Comment:
We should move this CSS to the website CSS file instead of having it inline here. The CSS file is in kafka-site, see https://github.com/apache/kafka-site/blob/asf-site/css/styles.css#L266.

Reply:
Ok, I'll add it to that file - was hoping to sneak past without having two separate pull requests ;)

I'll just add a class "config-list" and put everything in there?